### DISCUSSION TITLE
JSON output format change for `datetime` with V2?

### DISCUSSION THREAD

**Discussion by rassie** [pydantic/pydantic#11141] [Question] [answered]
This is currently not a problem, but I'm curious: with V2, the JSON output format for UTC `datetime`s seems to have changed from `2024-02-29T09:56:15+00:00` to `2024-02-29T09:56:15Z`. I assume this has something to do with aligning further with ISO8601, but was there a particular reason for this change, technical or otherwise?

**Comment by janas-adam** [comment, marked as answer]
hi,
its about [speedate](https://github.com/pydantic/speedate/?tab=readme-ov-file) which is datetime parsing logic for pydantic-core,
as speedate supports most common formats, the Z is preferred ~ (readability, compatibility, sticking to standards, performance etc.)

can be helpful:
[why-not-full-iso8601](https://docs.rs/speedate/latest/speedate/#why-not-full-iso8601)
[rfc3339-comparison-with-iso8601](https://ijmacd.github.io/rfc3339-iso8601/)
